src/Gql.js

```diff
@@ -229,12 +230,20 @@ class Gql {
 		return newResult;
 	}
 
-	static enum(val) {
-		return new GqlEnum(val);
+	static enum(name, val) {
+		return new GqlEnum(name, val);
 	}
 
-	enum(val) {
-		return this.constructor.enum(val);
+	static fragment(schema) {
+		return new GqlFragment({
+			name: schema.name,
+			type: schema.type,
+			fields: Schema.parseFragmentFields(schema.fields),
+		});
+	}
+
+	enum(name, val) {
+		return this.constructor.enum(name, val);
```

Comment:
we can use `this._enums` here.
If `val` is `undefined` then return from `this._enums`
otherwise create new instance of `GqlEnum`